core/src/main/java/io/github/mabartos/engine/LoginEventsEventListener.java

```diff
@@ -41,6 +41,12 @@ public void onEvent(Event event) {
     }
 
     protected void handleLogin(Event event) {
+        var riskEngine = session.getProvider(RiskEngine.class);
+        if (riskEngine != null && !riskEngine.isRiskBasedAuthnEnabled()) {
+            log.warn("Risk-based authentication is disabled for this realm.");
```

Comment:
I've added this log as WARN because I believe that if we disable risk based auth but we are still using dedicated flow , we should be warned about this. 
Maybe this is too much ?

Reply:
It'd be good to move this logic to the beginning of the onEvent() method. If the risk based authn is disabled, just ignore the events logging.

Moreover, in this case, I'd rather use the DEBUG level. The risk engine can be obtained from the session via the SPI mechanism, but does not have to be in any authentication flow.

Usually, you have specified the event listeners for the realm, and we can allow it to be there without any removals, but do nothing if the risk-based is disabled.

Reply:
Thanks for the comment. I agree ofc :) . Fix -> [1b8cbbd](https://github.com/mabartos/keycloak-adaptive-authn/pull/29/commits/1b8cbbdf7a06b08562f3071141463c08032bf548)